.github/workflows/labeler.yml

```diff
@@ -0,0 +1,16 @@
+# Automatically add labels to pull requests based on globs in the
+# .github/labeler.yml config file. For documentation see:
+# https://github.com/marketplace/actions/labeler
+---
+name: Pull Request Labeler
+
+on: [ pull_request_target ]
+
+jobs:
+  labeler:
+    permissions:
+      contents: read
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
I believe you first have to checkout the repo, so the labeler config file is accessible

// ref. https://github.com/actions/labeler?tab=readme-ov-file#using-configuration-path-input-together-with-the-actionscheckout-action

Reply:
The checkout is not required:

> The path to the label configuration file. If the file doesn't exist at the specified path on the runner, action will read from the source repository via the Github API.